src/Avalonia.Controls.DataGrid/DataGridRowGroupHeader.cs

```diff
@@ -430,6 +430,10 @@ internal void ToggleExpandCollapse(bool isVisible, bool setCurrent)
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value that indicates number format of items count
+        /// </summary>
+        public string CountFormat { get; set; } = "N1";
```

Comment:
make it a StyledProperty so you can customize via styles or ControlTheme

Reply:
thanks for sugestion, applied